corehq/apps/app_manager/views/modules.py

```diff
@@ -1128,7 +1133,10 @@ def edit_module_detail_screens(request, domain, app_id, module_unique_id):
         'long': params.get("long_custom_variables", None)
     }
     multi_select = params.get('multi_select', None)
-
+    auto_select = params.get('auto_select', None)
+    max_select_value = params.get('max_select_value', MULTI_SELECT_MAX_SELECT_VALUE)
```

Comment:
Nitpick: couldn't this be `max_select_value = params.get('max_select_value') or MULTI_SELECT_MAX_SELECT_VALUE` and then drop the if statement?

Reply:
ah yes indeed thank you! [8aa96ca](https://github.com/dimagi/commcare-hq/pull/32303/commits/8aa96ca6d362536cc1b2cd91f57d02bd4fcc628a)